tox.ini

```diff
@@ -27,9 +26,8 @@ setenv =
 
 [testenv:lint]
 skip_install = True
-basepython = python3.8
 commands =
-    black --check --diff .
+    - black --check --diff .
```

Comment:
I had to ignore the return code, as obviously this repos has not been blackened yet. Should I keep it this way or also blacken the repo? I would prefer to do this in a second PR.

Reply:
Agreed. Blackification should be in a separate PR, allowing this PR to focus on substantial changes.

Reply:
👍